### PR TITLE
fix message in sharetotwitter

### PR DESCRIPTION
**Pull request by upacyxou**
- sharetotwitter message changed

### CODE REVIEW

**Review by backmeupplz — APPROVED** [review]

File: src/components/badges/ShareToTwitterIfNeeded.tsx [hunk 1]
```diff
@@ -28,24 +37,48 @@ const wideBlock = classnames(
   gridColumn('lg:col-span-2', 'col-span-1')
 )
 
-export default function () {
+function ShareToTwitterIfNeededSuespended({
+  derivativeAddress,
+  network,
+}: {
+  derivativeAddress: string
```
Comment (T-Damer) [inline src/components/badges/ShareToTwitterIfNeeded.tsx:44]: Let's extract this props into `interface` they're same

File: src/components/badges/ShareToTwitterIfNeeded.tsx [hunk 2]
```diff
@@ -56,3 +89,20 @@ export default function () {
     </div>
   )
 }
+
+export default function ({
+  derivativeAddress,
+  network,
+}: {
+  derivativeAddress: string
```
Comment (T-Damer) [inline src/components/badges/ShareToTwitterIfNeeded.tsx:97]: Extract them into `interface ShareToTwitterProps`

Comment (upacyxou) [reply src/components/badges/ShareToTwitterIfNeeded.tsx]: fxied, ty

File: src/components/badges/ShareToTwitterIfNeeded.tsx (hunk 1 above)
Comment (upacyxou) [reply src/components/badges/ShareToTwitterIfNeeded.tsx]: fixed, ty

**Pull request merged**